build-support/bin/generate_github_workflows.py

```diff
@@ -75,7 +75,7 @@ def hashFiles(path: str) -> str:
 PYTHON37_VERSION = "3.7"
 PYTHON38_VERSION = "3.8"
```

Comment:
Stale?

Reply:
As mentioned in the PR description " a future change can clean up the CI scripts"... For now I just wanted to do the least invasive thing, until we're sure we don't need other interpreters for tests etc.